src/core-asyncpipeline/AsyncPipeline/AsyncPipeline.T/Equality/Equality.Equals.Wrap.cs

```diff
@@ -0,0 +1,12 @@
+#nullable enable
+
+namespace System
+{
+    partial struct AsyncPipeline<T>
+    {
+        public override bool Equals(object? obj)
```

Comment:
Prepare for .NET 6:
Change to `object? obj` to `[NotNullWhen(true)] object? obj`

Reply:
Done

Reply:
Done